src/forms/FormNav/FormNav.tsx

```diff
@@ -43,43 +64,46 @@ const FormNav = (props: FormNavProps): ReactElement => {
 	};
 
 	/**
-   * Goes to the sections that is selected.
-   * @param e
-   * @param newActiveTab
-   * @param sectionId
-   */
-	const handleClick = (e, newActiveTab, sectionId) => {
+	 * Goes to the sections that is selected.
+	 * @param e
+	 * @param idx
+	 * @param sectionId
+	 */
+	const handleClick = (e, idx: number) => {
 		e.preventDefault();
-		setSelectedTab(sectionId);
-		window.location.replace(`#${sectionId}`);
+		sectionsRefs[idx].scrollIntoView({ behavior: "smooth", block: "start", inline: "nearest"});
+		setSelectedTab(idx);
 	};
 
-	//IMPROVE RENDERING
-	let sectionsTest;
-
 	useEffect(() => {
-		sectionsTest = document.getElementsByClassName("section") as unknown as HTMLElement[];
-		window.addEventListener("scroll", navHighlighter);
+		const navHighlighter = () => {
+			const scrollY = window.pageYOffset;
+			if (sectionsRefs.length === 0) {
+				return;
+			}
 
-		return () => {
-			window.removeEventListener("scroll", navHighlighter);
+			sectionsRefs?.forEach(current => {
+				const currentSection = current as HTMLDivElement;
+				const sectionHeight = currentSection?.offsetHeight;
+				const sectionTop = currentSection?.offsetTop - 350;

```

Comment:
Not 100% sure if this is what might be causing the issue but the scrolling doesn't work on big screen, I tried using bigger sections but it doesn't seem to do anything... let's talk about it tomorrow.